src/aelfrice/detector_thresholds.py

```
@@ -0,0 +1,754 @@
+    "COVERED_WRITER_MODULES",
+    "DETECTOR_THRESHOLDS_VERSION",
+    "EXCLUDED_WRITERS",
+    "KINDS",
+    "DIGEST_HISTORY",
+    "MANIFEST_DIGEST",
+    "PinnedThreshold",
+    "THRESHOLDS",
+    "manifest_digest",
+    "pin_value",
+    "size_of",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)